airlock/login_api.py

```diff
@@ -0,0 +1,42 @@
+import json
+
+import requests
+from django.conf import settings
+
+
+class LoginError(Exception):
+    pass
+
+
+def get_user_data(user: str, token: str):
+    if settings.AIRLOCK_DEV_USERS_FILE and not settings.AIRLOCK_API_TOKEN:
+        return get_user_data_dev(user, token)
+    else:
+        return get_user_data_prod(user, token)
+
+
+def get_user_data_prod(user: str, token: str):
+    response = requests.post(
+        f"{settings.AIRLOCK_API_ENDPOINT}/releases/auth",
+        headers={"Authorization": settings.AIRLOCK_API_TOKEN},
+        json={"user": user, "token": token},
+    )
+    # We don't currently get any more detail about failures than this
```

Comment:
Hmm, yes, we could expose more info here, given its an authenticated by backend token.

I'm not sure the original decision to avoid exposing extra information was a good one any way, given the costs.